chefignore

```diff
@@ -90,7 +90,6 @@ Policyfile.lock.json
 # Cookbooks #
 #############
 CONTRIBUTING*
-CHANGELOG*
```

Comment:
changelog isn't in the chefignore anymore? 🤔

Reply:
See #20 was requested this way (not that stove seems to take chefignore into account ...)

Reply:
sorry about that, I could of sworn we had issues a while back with changelogs not getting pushed to our internal supermarket without taking out that line in the chefignore. Granted we were using knife supermarket share and not stove.